src/main/java/org/owasp/wrongsecrets/challenges/docker/Challenge34.java

```diff
@@ -0,0 +1,80 @@
+package org.owasp.wrongsecrets.challenges.docker;
+
+import java.util.List;
+import org.owasp.wrongsecrets.RuntimeEnvironment;
+import org.owasp.wrongsecrets.ScoreCard;
+import org.owasp.wrongsecrets.challenges.Challenge;
+import org.owasp.wrongsecrets.challenges.ChallengeTechnology;
+import org.owasp.wrongsecrets.challenges.Difficulty;
+import org.owasp.wrongsecrets.challenges.Spoiler;
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+
+/**
+ * This is a challenge based on leaking secrets with the misuse of Git notes
+ */
+@Component
+@Order(34)
+public class Challenge34 extends Challenge {
```

Comment:
Can you rename everything to Challenge35 please? Challenge34 is already in master...

Reply:
This is done now :)